package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "peerpad",
   "version": "0.3.1",
+  "homepage": "./",
   "private": true,
   "leadMaintainer": "Jim Pick <jim@jimpick.com>",
   "scripts": {
     "start": "node scripts/start.js",
-    "build": "GENERATE_SOURCEMAP=true GIT_COMMIT=$(git rev-parse HEAD) PUBLIC_URL='./' node scripts/build.js",
+    "build": "GIT_COMMIT=$(git rev-parse HEAD) node scripts/build.js",
```

Comment:
is removing the env vars intentional?

Reply:
Yes, they are:

- `GENERATE_SOURCEMAP` is used only once in `webpack.config.prod.js`:

```js
const shouldUseSourceMap = process.env.GENERATE_SOURCEMAP !== 'false';
``` 

It's default value is `true`, so there's no need to specify it there.

- `PUBLIC_URL` was removed in favor of the `homepage` property in the package.json as it is the recommend way to do it.

Reply:
@satazor gotcha, thank you!